install_files/ansible-base/roles/common/tasks/apt_upgrade.yml

```diff
@@ -58,7 +58,7 @@
   changed_when: tor_hold_package_result is changed
 
   # Ansible doesn't support notifying handlers based on file existence, results,
-  # e.g. via registered `stat` results, only only on changed=true task results,
+  # e.g. via registered `stat` results, only on changed=true task results,
```

Comment:
doubled words are _sometimes_ intentional. I don't think it's the case here, but...

Reply:
This certainly wasn't intentinal. 👍